aicsimageio/formats.py

```diff
@@ -194,4 +194,183 @@
     "lfr": "aicsimageio.readers.default_reader.DefaultReader",
     "lfp": "aicsimageio.readers.default_reader.DefaultReader",
     "spe": "aicsimageio.readers.default_reader.DefaultReader",
+    # Bioformats
+    # note: some of these should probably be above DefaultReader...
+    # but we need https://github.com/AllenCellModeling/aicsimageio/issues/307
```

Comment:
@toloudis I am fine with merging this and then fixing / updating when I make a PR for #307 on Friday